src/erlcloud_sqs.erl

```diff
@@ -261,7 +262,9 @@ decode_attribute_name("KmsMasterKeyId") -> kms_master_key_id;
 decode_attribute_name("KmsDataKeyReusePeriodSeconds") -> kms_data_key_reuse_period_seconds;
 decode_attribute_name("FifoQueue") -> fifo_queue;
 decode_attribute_name("DeduplicationScope") -> deduplication_scope;
-decode_attribute_name("FifoThroughputLimit") -> fifo_throughput_limit.
+decode_attribute_name("FifoThroughputLimit") -> fifo_throughput_limit;
+decode_attribute_name("SqsManagedSseEnabled") -> sqs_managed_sse_enabled;
+decode_attribute_name(Name) -> Name.
```

Comment:
not sure it's a good idea to fail later in request instead of here with func_clause...

Reply:
should we just leave these out?

Reply:
That is, filter out unknown attributes.

Reply:
Yeah, maybe this is a better way to do this. Want me to do this instead?

Reply:
IMO - leave only SqsManagedSseEnabled and not decode_attribute_name(Name)

Reply:
But that's what we had previously and now our services are having badtimes because of this new undocumented attribute :)

Reply:
I think we should leave it for now, handling the new attribute is enough to solve the problem. This is a fairly emergent issue so we should address it ASAP. Let's revisit how to handle unknown attributes in a follow-up. They definitely should be handled in some non-error/crash way end to end.